notebooks/116-sparsity-optimization/116-sparsity-optimization.ipynb

```diff
@@ -105,11 +108,12 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "id": "227d97eb-e91e-48bb-bba9-30598e49bb4f",
    "metadata": {},
    "source": [
-    "For benchmarking, we will use OpenVINO's benchmark application and let's put the IRs into a single folder."
+    "For benchmarking, you will use OpenVINO's benchmark application. Make sure to put the IRs into a single folder."
```

Comment:
probably meaning is different. The previous variant speaks that code below put all files in the same directory, current version speaks that the user is responsible for downloading and should check that all files will be downloaded in the same folder.

Reply:
How about: 

```suggestion
    "For benchmarking, you will use OpenVINO's benchmark application. Use the code below to put the IRs into a single folder."
```